template.tpl

```diff
@@ -0,0 +1,1704 @@
+  },
+  "description": "A Google Tag Manager Server-side Tag template for sending events to any JSON HTTP endpoint. Works best with the Snowplow Client.",
+  "containerContexts": [
+    "SERVER"
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "type": "TEXT",
+    "name": "url",
+    "displayName": "Destination URL",
+    "simpleValueType": true,
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ],
+    "valueHint": "https://",
+    "alwaysInSummary": true
+  },
+  {
+    "type": "CHECKBOX",
+    "name": "useDotNotation",
+    "checkboxText": "Use dot notation",
+    "simpleValueType": true,
+    "help": "Disable this option to disallow the use of dot notation to create a nested request object.",
+    "defaultValue": true
+  },
```

Comment:
This is literally my favourite feature 👏

What would be a use case where you would ever want to disable it?

Reply:
There may be the case that maybe a property name like `event.data` may be desired. 
I.e. 
```
{
  "event.data": "xyz"
}
```
instead of
```
{
  "event": {
    "data": "xyz"
  }
}
```

However i am not sure how usual that would be. Also, it is an all-or-nothing disablement of the feature. I saw the idea from the JSON HTTP Request tag, so i assumed there may be use cases like this. Do you think it makes sense?

Reply:
Thats a good point, I think my fear is you'd want to mix and match rather soon...

```
{
  "event.data": {
    "myarray": [
        "one", "two"
    ]
  }
}
```

I feel we should be simply always supporting dot notation but we can support escape special characters like `.` in a future release.

My suggestion would be to remove this check box and always support dot notation for the 0.1.0 release. Later we can work on escaping the value like this perhaps: https://stackoverflow.com/questions/19726859/jsonpath-junit-escape-character-for-dots
Or figure out what the getEventData dot notation uses for escaping with GTM and stay consistent in the GTM world.